test/static/reject/ufuncs.pyi

```diff
@@ -84,3 +85,9 @@ np.rad2deg(AR_f8, dtype=np.datetime64)  # type: ignore[arg-type]  # pyright: ign
 
 np.radians(dt64)  # type: ignore[call-overload]  # pyright: ignore[reportCallIssue,reportArgumentType]
 np.radians(AR_f8, dtype=np.datetime64)  # type: ignore[arg-type]  # pyright: ignore[reportArgumentType, reportCallIssue]
+
+np.copysign(i8, i8, dtype=np.datetime64)  # type: ignore[arg-type]  # pyright: ignore[reportCallIssue,reportArgumentType]
```

Comment:
This way, it's a bit more consistent with the ones below (although I see that it's consistently inconsistent above here, but that's out-of-scope right now)

```suggestion
np.copysign(i8, i8, dtype=np.datetime64)  # type: ignore[arg-type]  # pyright: ignore[reportArgumentType, reportCallIssue]
```